app/src/main/res/layout/bottom_sheet_nearby.xml

```diff
@@ -38,7 +38,7 @@
     </android.support.constraint.ConstraintLayout>
 
     <FrameLayout
-        android:id="@+id/container_sheet"
+            android:id="@+id/container_sheet"
```

Comment:
Could you please submit a new pull request without indentation/spacing changes? If you want to submit indentation/spacing changes it is OK but please do it in another pull request. Solving only one issue per pull request makes review much easier.
Thanks a lot! :-)

Reply:
Hey, @nicolas-raoul I am not able to create a new pull request!